### PR TITLE
Adds butt shoes

### PR DESCRIPTION
**Pull request by PopNotes**
:cl: PopNotes
add: Added butt shoes. I'm sorry.
/:cl:

Butt shoes. They fart when you walk.
Also fixes an internal typo, just to get that one nice and squared off.

### CODE REVIEW

**Comment by Swagile** [comment]
would it not be better to make butt shoes a child of clown shoes and make them fart just like clown shoes do in the code instead of adding a component in the initialization of the butt shoes?

**Comment by PopNotes** [comment]
I would have to do this either way

**Comment by Swagile** [comment]
> byond code

why u heff to b so patchworky

**Comment by GuyonBroadway** [comment]
pics of the sprits pls.

**Comment by PopNotes** [comment]
wew

**Comment by GuyonBroadway** [comment]
oh my god they are fucking cloggs.

**Comment by GuyonBroadway** [comment]
BUTT CLOGS!

**Comment by PopNotes** [comment]
wait fuck the crafting recipe doesn't work
hold it

**Comment by PopNotes** [comment]
aight fixed it

File: hippiestation/code/modules/clothing/shoe.dm [hunk 1]
```diff
@@ -1,3 +1,17 @@
+/obj/item/clothing/shoes/buttshoes
+	desc = "Why?"
+	name = "butt shoes"
+	alternate_worn_icon = 'hippiestation/icons/mob/feet.dmi'
+	icon = 'hippiestation/icons/obj/clothing/shoes.dmi'
+	icon_state = "buttshoes"
+	item_state = "buttshoes"
+	item_color = "buttshoes"
+	slowdown = SHOES_SLOWDOWN+1
```
Comment (ghost) [inline hippiestation/code/modules/clothing/shoe.dm]: NO SLOWDOWN PLZ

Comment (PopNotes) [reply hippiestation/code/modules/clothing/shoe.dm]: fuckin

**Comment by PopNotes** [comment]
RATE THIS REEEEEEEE

**Comment by GuyonBroadway** [comment]
CONFLIGTS!!

**Comment by PopNotes** [comment]
GNO!

**Comment by PopNotes** [comment]
what the fuck is conflicting

**Pull request closed without merging**